### ISSUE TITLE
Firefox openes tab with /notifications/sync

### ISSUE THREAD

**Issue by bitboxer** [octobox/octobox#555]
Are you experiencing this issue in octobox.io or your own instance?
 - [ ] octobox.io
 - [X] My instance

🐞 **Problem**

When I restart my Firefox and it reloads the previews opened tabs it always opens octobox with `/notifications/sync` as path. Sadly that path does not exist as get and it fails to load the page.

🎯 **Goal**

It should try to load `/` when Firefox restarts.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by chrisarcand** [comment]
[`notifications/sync` does exist as a `GET` route](https://github.com/octobox/octobox/blob/e38a435b4081f2038f2a794222ded27bd3f9feae/config/routes.rb#L18), so you can just refresh the page without removing the slug. Leaving this open though as I'm still unsure why Octobox falls on itself in a long running tab, though I suspect it has something to do with auto-refresh when you don't have an internet connection or something.

**Comment by bitboxer** [comment]
Yes, it's a bit weird. I will keep an eye on it and will try to find out why this happens.

**Comment by bitboxer** [comment]
Okay, happened again. I opened my Mac this morning and it took it some time to get wifi. In that time the firefox tried to reload the page and failed. When I press Cmd+R to reload the page it tries to do a PUT request and says it can't find the page. Does the application refresh the page with a PUT instead of a GET?

**Comment by andrew** [comment]
I've noticed this happen on mobile as well, pretty annoying!

**Comment by jules2689** [comment]
> Does the application refresh the page with a PUT instead of a GET?

Most likely. When I added the `GET` endpoint it was to fix the issue where the page would fail to load because when you leave a long running chrome tab opens it loses context and falls back to `GET`.

I think a solution here would be to change to use the new sync job (unless we are already) and remove the `PUT` endpoint, using only `GET`

**Comment by Alexkho82** [comment]
#555

**Comment by andrew** [comment]
@bitboxer I suspect this is because CMD+R is triggering a sync at the same time as refreshing the page

We could also redirect GET requests to `/notifications/sync` back to `/`

**Comment by bitboxer** [comment]
Yes, that could work.

**Comment by andrew** [comment]
@bitboxer is this still happening for you? There have been quite a few changes related to syncing that I believe should fix this.

**Comment by bitboxer** [comment]
Updating my instance now and will try it out for a couple of days. Reporting back on Saturday if this worked.

**Comment by pradyunsg** [comment]
I can confirm that this hasn't occurred for me since I enabled background jobs.

**Comment by bitboxer** [comment]
@pradyunsg what do you mean with enabling background jobs?

**Comment by pradyunsg** [comment]
https://github.com/octobox/octobox/blob/master/docs/INSTALLATION.md#background-jobs

**Comment by bitboxer** [comment]
Okay, tested it now with the cases where I had the problem and it disappeared. Thanks! Closing this now.

**Comment by bitboxer** [comment]
Sadly this is happening again. My firefox opens `/notifications/sync` after I wake it up from sleep. The page just says `We've had a problem at our end.`. So the redirect back to the root is not there anymore.
